src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,29 @@
+/*Class created by Priyanka Agarwal as per code written
+on codeU directions. Only added package and import statements
+ */
+
+package codeu.chat.common;
+import codeu.chat.util.Uuid;
+import java.io.IOException;
+
+
+public final class ServerInfo {
+    private final static String SERVER_VERSION = "1.0.0";
```

Comment:
Maybe a comment or documentation somewhere saying that this variable should be updated whenever the server code changes?